plugins/gitops-profiles/src/components/ProfileCatalog/ProfileCatalog.test.tsx

```diff
@@ -31,6 +33,9 @@ import { githubAuthApiRef } from '@backstage/core-plugin-api';
 describe('ProfileCatalog', () => {
   it('should render', async () => {
     const oauthRequestApi = new OAuthRequestManager();
+    const configApi: ConfigApi = new ConfigReader({
+      enableExperimentalRedirectFlow: false,
+    });
```

Comment:
```suggestion
    const configApi = new MockConfigApi({});
```

I'm feeling it's a bit noisy to have the `false` here in all the tests, this should be enough. By using `MockConfigApi` from test utils we can avoid the extra dependency as well, and is overall a bit cleaner.